force-app/main/default/classes/FlowControlLoopsCollections.cls

```diff
@@ -162,17 +217,16 @@ public with sharing class FlowControlLoopsCollections {
      */
     public static String repeatString(String inputString, Integer repeatCount) {
         // Initialize the result String
-
-        // Use a for loop to append the inputString to the result string repeatCount times
-        /*
-        for () {
-
-            // If it is not the last iteration, add a semicolon to separate the strings
-        }
-        */
-
-        // Return the final result string
-        return null; // Replace null with the variable you used to store the result
+        String sc = ';';
+        String result = '';
+        for (Integer r = 1; r <= repeatCount; r++) {
+           if (r < repeatCount) {
+            result += inputString + sc + ' ';
+           } else {
+             result += inputString;
```

Comment:
Well done. You can add the empty space after the ';' in your sc String so that you don't need to add it each time on line 224